### ISSUE TITLE
helmfile delete doesn't use the specified tillerNamespace

### ISSUE THREAD

**Issue by irlevesque** [roboll/helmfile#209]
- helmfile version v0.21.0

```
→ head -2 helmfile.d/0100.app.yaml
helmDefaults:
  tillerNamespace: {{ env "DEPLOY_ENV" }}
```

`sync` works as expected via

```
DEPLOY_ENV=staging helmfile -l chart=app sync
```

`delete` complains:

```
→ DEPLOY_ENV=staging helmfile --log-level debug -l chart=app delete
Deleting app-staging
exec: helm delete app-staging
Error: could not find tiller

err: exit status 1
```

**Comment by mumoshu** [comment]
Thanks for reporting!
This should be fixed by #201. The PR has been already merged but not yet released. I'll release v22.0 asap.

**Comment by mumoshu** [comment]
v22.0 has been already released with the fix for this!
